src/main/webapp/app/shared/utils/Utils.tsx

```diff
@@ -272,7 +272,7 @@ export function getDefaultColumnDefinition<T>(
         Header: <span>Tumor Type</span>,
         accessor: 'cancerType',
         style: { whiteSpace: 'normal' },
-        minWidth: 150,
+        minWidth: 250,
```

Comment:
Why are we changing the minWidth?

Reply:
Because some dx/px tumor types are too long to be displayed in the table column. For example, `B-Lymphoblastic Leukemia/Lymphoma with t(9;22)(q34.1;q11.2);BCR-ABL1` will be displayed as two lines. It looks not good. Changing minWidth here could make every row in table has the same format.

Reply:
Overwrite the minWidth at the place you use this column definition.